examples/docs-examples/examples/transferring-data/upload-file.mjs

```diff
@@ -11,18 +11,22 @@ import * as fs from "fs";
   const buff = fs.readFileSync("worker.mjs");
   const hash = createHash("md5").update(buff).digest("hex");
 
-  const result = await executor.run(async (ctx) => {
-    await ctx.uploadFile("./worker.mjs", "/golem/input/worker.mjs");
+  try {
+    const result = await executor.run(async (ctx) => {
+      await ctx.uploadFile("./worker.mjs", "/golem/input/worker.mjs");
 
-    const res = await ctx.run(
-      `node -e "const crypto = require('node:crypto'); const fs = require('fs'); const buff = fs.readFileSync('/golem/input/worker.mjs'); const hash = crypto.createHash('md5').update(buff).digest('hex'); console.log(hash); "`,
-    );
+      const res = await ctx.run(
+        `node -e "const crypto = require('node:crypto'); const fs = require('fs'); const buff = fs.readFileSync('/golem/input/worker.mjs'); const hash = crypto.createHash('md5').update(buff).digest('hex'); console.log(hash);"`,
+      );
 
-    return res.stdout;
-  });
-
-  console.log("md5 of the file sent to provider: ", result);
-  console.log("Locally computed  md5: ", hash);
+      return res.stdout;
+    });
 
-  await executor.end();
+    console.log("md5 of the file sent to provider: ", result);
+    console.log("Locally computed md5: ", hash);
+  } catch (error) {
+    console.error("Computation failed:", error);
```

Comment:
I like that you provide context in this log message - that's a good practice. Could you please include it in other examples you modified in this PR?